client/src/components/profile/profile.tsx

```diff
@@ -17,50 +17,28 @@ interface ProfileProps {
   user: User;
 }
 
-function renderMessage(
-  isSessionUser: boolean,
-  username: string,
-  t: TFunction
-): JSX.Element {
-  return isSessionUser ? (
-    <>
-      <FullWidthRow>
-        <h2 className='text-center'>{t('profile.you-not-public')}</h2>
-      </FullWidthRow>
-      <FullWidthRow>
-        <p className='alert alert-info'>{t('profile.you-change-privacy')}</p>
-      </FullWidthRow>
-      <Spacer size='medium' />
-    </>
-  ) : (
-    <>
-      <FullWidthRow>
-        <h2 className='text-center' style={{ overflowWrap: 'break-word' }}>
-          {t('profile.username-not-public', { username: username })}
-        </h2>
-      </FullWidthRow>
-      <FullWidthRow>
-        <p className='alert alert-info'>
-          {t('profile.username-change-privacy', { username: username })}
-        </p>
-      </FullWidthRow>
+function Message({ t }: { t: TFunction }): JSX.Element {
```

Comment:
the reason that we are removing the `props` because Message without the props are never used, so I have used the one that is always true

Reply:
I might be missing something, but if the user have an active session they are addressed directly:
"you-not-public": "You have not made your portfolio public.",
"you-change-privacy": "You need to change your privacy setting in order for your portfolio to be seen by others. This is a preview of how your portfolio will look when made public.",

And if they is not active session: 
"username-not-public": "{{username}} has not made their portfolio public.",
"username-change-privacy": "{{username}} needs to change their privacy setting in order for you to view their portfolio.",

Since we are not checking for isSessionUser, shouldn't both versions be included?

If you decide to move forward the version with no active session, "username-not-public", and "username-change-privacy" should be deleted as they are not used anywhere else.